src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -217,6 +217,12 @@ public boolean isExternalSubsParsed() {
 	private byte[] h264_annexB;
 
 	/**
+	 * @deprecated Use standard getter and setter to access this variable.
+	 */
+	@Deprecated
+	public String truncated;
+
+	/**
 	 * Not stored in database.
```

Comment:
OK, i get it. Thank you for the link and explaination :smile: 
Just because i become suspicious, you don't work on that, right ?! Because i didn't asked for, and i don't think it's should worst of it. I'll play on it though.
Sometimes i feel like you understand "advices" as "work on it", and after i feel bad, so i prefer just cut it done on it now.

Nothing is for "ever" in this universe, the life, existing law is change/evolution or death.
If i remember correctly, @SubJunk have told about cleaning all deprecated, unused stuff in the code, but from what i remember he was afraid to broke plugins.

From what i understood from @SharkHunter , only Jumpy and Channels are still working, and he say that they are near the same peace of code, so as we have @valib using Jumpy and @SharkHunter using Channels regulary, if something came to break they will tell us fast.
So I don't see the end of the world, that's said are there are popular plugins, for sure about Jumpy, i think that compatibility should be preserved, and with their help, feedback that should be doable. That's only my 2 cents though...

**P.S.** To be honest the "package private" will still stay mysterious to me for quit sometime ;)